frontend/src/components/grant-form/index.tsx

```diff
@@ -66,12 +65,12 @@ export const GrantForm = ({ conference }: Props) => {
       formState.setField("fullName", user.fullName);
       formState.setField("name", user.name);
       formState.setField("gender", user.gender);
-      if (user.dateBirth) {
-        formState.setField(
-          "age",
-          new Date().getFullYear() - new Date(user.dateBirth).getFullYear(),
-        );
-      }
+      // if (user.dateBirth) {
```

Comment:
do we want to automatically pre-select the age group?

Reply:
yes I was thinking about how to do it in an elegant way... :D